extensions/wikia/AdEngine/js/SlotTweaker.js

```diff
@@ -1,4 +1,4 @@
-/*global define*/
```

Comment:
I would break SlotTweaker into smaller, manageable modules. This way those could be unit tested. I understand that we have deadline. How about creating the subtask to merge the solution from this PR and another one to do some refactor in the SlotTweaker area?